packages/mobile_app/lib/screens/registration/accept_terms.dart

```diff
@@ -16,58 +17,52 @@ class AcceptTermsRow extends StatefulWidget {
 class _AcceptTermsRowState extends State<AcceptTermsRow> {
```

Comment:
This widget is getting more and more ugly :)
Have you tried [RichText](https://api.flutter.dev/flutter/widgets/RichText-class.html)? Maybe it become better over time?

Reply:
Why? You just take out right column items as final part and actual widget looks and loads better:

```
ListView(
        shrinkWrap: true,
        scrollDirection: Axis.horizontal,
        physics: NeverScrollableScrollPhysics(),
        children: <Widget>[
          MagicCheckbox(
            value: widget.terms,
            onChanged: (_) {},
          ),
          const SizedBox(
            width: 16.0,
          ),
          //
          Column(
            crossAxisAlignment: CrossAxisAlignment.start,
            mainAxisAlignment: MainAxisAlignment.center,
            children: **rightColumnItems**,
          ),
        ],
      ),
```